### DISCUSSION TITLE
Is there a way to use htmx in wordpress?

### DISCUSSION THREAD

**Discussion by stratboy** [bigskysoftware/htmx#1163] [Q&A]
Hi, I'm curious of this "html over the wire" approach. Though, The 99% of my job is building 100% wordpress custom thems for my clients. I see htmx, unpoly and especially hotwire seem mainly used with web app frameworks like Rails. 
Is there a way to implement the in wordpress?  At least for just the main content?

**Comment by gone** [comment]
Check out the hx-boost extension, which might fit your case well (https://htmx.org/attributes/hx-boost/)

**Comment by jasalt** [comment]
Htmxpress plugin gives good a starting point https://github.com/svandragt/htmxpress. I have been building some time interactive html with it last couple months and it's working pretty much as expected. Not in production yet but haven't faced major issues. Seems to work fine with multi-site also.

Made this example for usage as a shortcode https://github.com/jasalt/htmxpress-serverside-shortcode and there is a block example also https://github.com/svandragt/htmxpress-serverside-block/ which needs some work to make forms play nice in block editor.
